jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ContextHandler.java

```diff
@@ -2607,6 +2616,8 @@ protected void doStop() throws Exception
         @Override
         public void insertHandler(Singleton handler)
         {
+            // We cannot call super.insertHandler here, because it uses this.setHandler
+            // which gives a warning.  This is the same code, but uses super.setHandler
```

Comment:
See above about not issuing a warning, but defaulting to setting the "tail".

Also, why `setHandler()` below is overridden to throw?

Reply:
@sbordet Because setting the handler of the coreContextHandler breaks this class as it removes the core2nestedHandler.      Inserting is OK, but replacing is not.